web/docs/faqs.md

```diff
@@ -63,21 +71,24 @@ with the [Sovereign Edition](https://tenzir.com/pricing).
 ## Does Tenzir run on-premise?
 
 Yes, Tenzir can run on premise and support fully air-gapped environments. The
-[Sovereign Edition](https://tenzir.com/pricing) allows you to deploy the entire
-platform in a dockerized environment, such as Docker Compose.
+[Sovereign Edition](https://tenzir.com/pricing) allows you to [deploy the entire
+platform](installation/deploy-the-platform.md) in a dockerized environment, such
```

Comment:
```suggestion
platform](installation/deploy-the-platform.md) in a Dockerized environment, such
```

Reply:
Huh, really? Isn't this an adjective and therefore lowercase?

Reply:
Hmm, my intuition said that it should be uppercase because it's created from a company name. But note sure what takes precedence here. Google finds plenty hits for both.

Feel free to just resolve, but if you do happen to find out which is correct and why please let me know. 😅

Reply:
ChatGPT agrees with you:
> “Dockerized” is capitalized because it derives from Docker, which is a proper noun.